README.md

```diff
@@ -47,8 +47,9 @@ http {
                 -- Please set them as service logic name or DNS name
                 --
                 -- Currently, we can not have the upstream real network address
+                -- The parameter "customer_namespace" for java agent config "agent.namespace"
                 ------------------------------------------------------
-                require("tracer"):start("upstream service")
+                require("tracer"):start("upstream service", "customer_namespace")
```

Comment:
This key should only be a optional parameter.

Reply:
Yes, this parameter will be used only after we use the configuration of “agent.namespace” in the java agent. if we don't use it , this parameter can be nil